vm-setup/roles/v1aX_integration_test/templates/controlplane_ubuntu.yaml

```diff
@@ -24,12 +24,17 @@ spec:
           provider-id: 'metal3://{{ '{{ ds.meta_data.uuid }}' }}'
     clusterConfiguration:
       apiServer:
-        extraArgs:
+        extraArgs:      
+    users:
+    - name: metal3
+      sudo: "ALL=(ALL) NOPASSWD:ALL"
+      sshAuthorizedKeys:
+      - {{ SSH_PUB_KEY_CONTENT }}          
     preKubeadmCommands:
       - ip link set dev enp2s0 up
       - dhclient enp2s0
-      - mv /tmp/akeys /home/ubuntu/.ssh/authorized_keys
-      - chown ubuntu:ubuntu /home/ubuntu/.ssh/authorized_keys
+      - mv /tmp/akeys /home/metal3/.ssh/authorized_keys
+      - chown metal3:metal3 /home/metal3/.ssh/authorized_keys
```

Comment:
same here, and in the files section below

Reply:
Done